.github/workflows/build_linux.yml

```diff
@@ -14,9 +14,20 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  handle-syncwith:
+    if: github.event_name == 'pull_request'
+    name: Call Reusable SyncWith Handler
+    uses: NilFoundation/ci-cd/.github/workflows/reusable-handle-syncwith.yml@0588663776c5989c2ba23064811e7f71a20ada83
+    with:
+      ci-cd-ref: '0588663776c5989c2ba23064811e7f71a20ada83'
```

Comment:
Do we need this commit hashes here?

Reply:
This way we won't brake current CI even with bad commit merged into ci-cd repo. If we will frequently modify this, we should better replace it with branch and make that branch protected

Reply:
Maybe we could use a tag instead?

Reply:
I replaced SHAs with tags to make it more pleasing to the eye, but we will have to update it manually anyway